test/run_pass/issue/issue-133.js

```diff
@@ -13,6 +13,14 @@
  * limitations under the License.
  */
 
+if (this.driver) {
```

Comment:
If this code is valid for every testfiles, how about replacing it before & after evaluating every file contents in test driver?
(Object.defineProperty() with writable=false might be helpful, if you're worried about